docs/_data/table/master-configuration.csv

```diff
@@ -4,7 +4,7 @@ alluxio.master.heartbeat.interval.ms,1000
 alluxio.master.hostname,localhost
 alluxio.master.file.async.persist.handler,alluxio.master.file.async.DefaultAsyncPersistHandler
 alluxio.master.format.file_prefix,""_format_""
-alluxio.master.journal.folder,${alluxio.home}/journal/
+alluxio.master.journal.folder,${alluxio.work.directory}/journal
```

Comment:
We're trying to switch to directory wherever possible to be consistent. We could add an alluxio.master.journal.directory parameter with default ${alluxio.master.journal.folder} and replace the docs for .folder with .directory - if that sounds good to you I will do it in a followup PR